common/interpreter/wawaka_wasm/WawakaInterpreter.h

```diff
@@ -38,6 +38,7 @@ class WawakaInterpreter : public pc::ContractInterpreter
     wasm_module_t wasm_module = NULL;
     wasm_module_inst_t wasm_module_inst = NULL;
     wasm_exec_env_t wasm_exec_env = NULL;
+    ByteArray binary_code;
 
```

Comment:
in theory, all of these should have a "_" appended to the end for our coding standards

Reply:
Can we make these changes in a separate PR, since this is about coding formatting?

Reply:
for the moment, just fix the one.